### PR TITLE
Suse theme liberty

### PR DESCRIPTION
**Pull request by toabctl**

### CODE REVIEW

**Comment by aplanas** [comment]
Can this be merge before Liberty packages arrives?  We can break M6

**Comment by toabctl** [comment]
> Can this be merge before Liberty packages arrives? We can break M6

No. We need to wait for Liberty packages. We also need to adjust the package to install the new directory.

**Comment by toabctl** [comment]
@aplanas can be merged now. I switch the package in D:C:6 to track the kilo branch. So master is open for Liberty now.

File: themes/suse/_variables.scss [hunk 1]
```diff
@@ -0,0 +1 @@
+@import "../themes/default/variables";
```
Comment (aplanas) [inline themes/suse/_variables.scss]: nitpick: new line at the end of the scss file?

**Comment by aplanas** [comment]
+1 for me

**Comment by sayalilunkad** [comment]
+1 LGTM

**Comment by vuntz** [comment]
Assuming it works, +1

**Comment by toabctl** [comment]
Well - it works but needs some package modifications and also some chef cookbook modification. Using the theme needs now 2 steps:
- change the CUSTOM_THEME var in local_settings.py to the installed theme
- then run the collectstatic management command to collect the custom theme static files

imo that' worse than before but it seems to be the way what upstream suggests...

**Pull request merged**